dbt/include/redshift/macros/metadata/relation_last_modified.sql

```diff
@@ -0,0 +1,29 @@
+{% macro redshift__get_relation_last_modified(information_schema, relations) -%}
+
+    {%- call statement('last_modified', fetch_result=True) -%}
+        select
+            ns.nspname as "schema",
+            c.relname as identifier,
+            max(qd.start_time) as last_modified,
+            {{ current_timestamp() }} as snapshotted_at
+        from pg_class c
+        join pg_namespace ns
+            on ns.oid = c.relnamespace
+        join sys_query_detail qd
+            on qd.table_id = c.oid
+        where qd.step_name in ('insert', 'merge', 'mergejoin')
```

Comment:
Does this cover all scenarios?

Reply:
hmm I actually wonder if 'merge' and 'mergejoin' should be excluded: https://docs.aws.amazon.com/redshift/latest/dg/r_EXPLAIN.html

Looks like the only "step" that modifies a table is 'insert'

Reply:
Good catch, that's what I get for assuming. I thought that was referencing a "merge" into an existing table, like with `pandas`.